services/dsp-b/src/index.ts

```diff
@@ -201,6 +202,20 @@ app.get('/bidding_signal.json', async (req: Request, res: Response) => {
   });
 });
 
+app.get('/header-bid', async (req, res) => {
+  res.json({
+    bid: Math.floor(Math.random() * 50),
```

Comment:
In dsp-a bidding-logic.js you choose " bid: Math.floor(Math.random() * 100, 10)," 
Is there a reason for changing the calculation of the bid value here?

Reply:
Yes, I've set those values to allow more PA auctions to win than contextual auctions since this is a PA demo. With the values set to 0-100 for both header bidding auction and ad server auction caused a lot of contextual auction ads to win, so I've reduces the probability that a contextual auction would win.  

I'll explain this in the comments.